pkg/component/kubernetes/controllermanager/controllermanager_test.go

```diff
@@ -202,98 +173,40 @@ var _ = Describe("KubeControllerManager", func() {
-								"role": "controller-manager",
-							},
-						},
-						Template: hvpav1alpha1.HpaTemplate{
-							ObjectMeta: metav1.ObjectMeta{
-								Labels: map[string]string{
-									"app":  "kubernetes",
-									"role": "controller-manager",
-								},
-							},
-							Spec: hvpav1alpha1.HpaTemplateSpec{
-								MinReplicas: ptr.To[int32](1),
-								MaxReplicas: int32(1),
-							},
-						},
+					UpdatePolicy: &vpaautoscalingv1.PodUpdatePolicy{
+						UpdateMode: &vpaUpdateMode,
 					},
-					Vpa: hvpav1alpha1.VpaSpec{
-						Selector: &metav1.LabelSelector{MatchLabels: map[string]string{
-							v1beta1constants.LabelRole: "kube-controller-manager-vpa",
-						}},
-						Deploy: true,
-						ScaleUp: hvpav1alpha1.ScaleType{
-							UpdatePolicy: hvpav1alpha1.UpdatePolicy{
-								UpdateMode: ptr.To(hvpav1alpha1.UpdateModeAuto),
-							},
-						},
-						ScaleDown: hvpav1alpha1.ScaleType{
-							UpdatePolicy: hvpav1alpha1.UpdatePolicy{
-								UpdateMode: scaleDownUpdateMode,
+					ResourcePolicy: &vpaautoscalingv1.PodResourcePolicy{
+						ContainerPolicies: []vpaautoscalingv1.ContainerResourcePolicy{{
+							ContainerName: "kube-controller-manager",
+							MinAllowed: corev1.ResourceList{
+								corev1.ResourceMemory: resource.MustParse("50Mi"),
 							},
-						},
-						Template: hvpav1alpha1.VpaTemplate{
-							ObjectMeta: metav1.ObjectMeta{
-								Labels: map[string]string{
-									v1beta1constants.LabelRole: "kube-controller-manager-vpa",
-								},
-							},
-							Spec: hvpav1alpha1.VpaTemplateSpec{
-								ResourcePolicy: &vpaautoscalingv1.PodResourcePolicy{
-									ContainerPolicies: []vpaautoscalingv1.ContainerResourcePolicy{{
-										ContainerName: "kube-controller-manager",
-										MinAllowed: corev1.ResourceList{
-											corev1.ResourceMemory: resource.MustParse("50Mi"),
-										},
-										MaxAllowed: corev1.ResourceList{
-											corev1.ResourceCPU:    resource.MustParse("4"),
-											corev1.ResourceMemory: resource.MustParse("10G"),
-										},
-										ControlledValues: &controlledValues,
-									}},
-								},
+							MaxAllowed: corev1.ResourceList{
+								corev1.ResourceCPU:    resource.MustParse("4"),
+								corev1.ResourceMemory: resource.MustParse("10G"),
 							},
-						},
-					},
-					WeightBasedScalingIntervals: []hvpav1alpha1.WeightBasedScalingInterval{
-						{
-							VpaWeight:         hvpav1alpha1.VpaOnly,
-							StartReplicaCount: 1,
-							LastReplicaCount:  1,
-						},
-					},
-					TargetRef: &autoscalingv2beta1.CrossVersionObjectReference{
-						APIVersion: appsv1.SchemeGroupVersion.String(),
-						Kind:       "Deployment",
-						Name:       "kube-controller-manager",
+							ControlledValues: &controlledValues,
+						}},
 					},
 				},
 			}
+
+			if isScaleDownDisabled {
+				vpa.Labels = map[string]string{v1beta1constants.LabelVPAEvictionRequirementsController: v1beta1constants.EvictionRequirementManagedByController}
```

Comment:
You could also follow the pattern here in which we explicitly specify the label and annotation keys and values instead of using constants.

Reply:
Done.